### PR TITLE
Add tooltip directive, overlay service, partial

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Open tooltip on Edit Schedule click
Handle closing tooltip and overlay

[stage-2]

## Motivation and Context
Refactoring and code improvements. Getting closer to final functionality.

TODOs:
- Tooltip and overlay open/close based on localStorage logic
- Styling for the tooltip html
- Mobile view
- Tests

## How Has This Been Tested?
Tested changes locally.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
See todos list

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: web/scripts/common/directives/dtv-tooltip-overlay.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+angular.module('risevision.apps.directives')
+  .directive('tooltipOverlay', ['$compile', '$timeout', 'honeBackdropFactory',
+    function ($compile, $timeout, honeBackdropFactory) {
+      return {
+        restrict: 'A',
+        scope: {
+          isShowing: '='
+        },
+        terminal: true,
```
Comment (ezequielc) [inline web/scripts/common/directives/dtv-tooltip-overlay.js:11]: @alex-deaconu Curious, what happens if we don't set `terminal: true` here? Thanks!

Comment (alex-deaconu) [reply web/scripts/common/directives/dtv-tooltip-overlay.js:11]: @ezequielc good question. I think it's best you review this for a thorough explanation of the code: https://stackoverflow.com/questions/19224028/add-directives-from-directive-in-angularjs. Thanks.

Comment (ezequielc) [reply web/scripts/common/directives/dtv-tooltip-overlay.js:11]: @alex-deaconu That clarifies it. Thanks!

**Pull request merged**